obspy/io/sh/core.py

```diff
@@ -27,6 +27,9 @@
 MONTHS = ['JAN', 'FEB', 'MAR', 'APR', 'MAY', 'JUN', 'JUL', 'AUG', 'SEP',
           'OCT', 'NOV', 'DEC']
 
+MONTHS_DE = ['JAN', 'FEB', 'MAR', 'APR', 'MAI', 'JUN', 'JUL', 'AUG', 'SEP',
+             'OKT', 'NOV', 'DEZ']
```

Comment:
I'd switch to use a dictionary, that could combine everything and the lookup done below wouldn't be as ugly with the `.index() + 1` call..

```python
MONTHS = {
    ...
    'DEC': 12,
    'DEZ': 12,
    }

Reply:
Nice idea, but I'd rather stick with the current state, because

1. it would need some extra logic to define the default for writing in `from_utcdatetime`
2. the German names are a fallback and the try except clause makes this clear